merge_requests.go

```diff
@@ -403,12 +403,17 @@ func (s *MergeRequestsService) GetIssuesClosedOnMerge(pid interface{}, mergeRequ
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/merge_requests.html#create-mr
 type CreateMergeRequestOptions struct {
-	Title           *string `url:"title,omitempty" json:"title,omitempty"`
-	Description     *string `url:"description,omitempty" json:"description,omitempty"`
-	SourceBranch    *string `url:"source_branch,omitempty" json:"source_branch,omitempty"`
-	TargetBranch    *string `url:"target_branch,omitempty" json:"target_branch,omitempty"`
-	AssigneeID      *int    `url:"assignee_id,omitempty" json:"assignee_id,omitempty"`
-	TargetProjectID *int    `url:"target_project_id,omitempty" json:"target_project_id,omitempty"`
+	Title              *string `url:"title,omitempty" json:"title,omitempty"`
+	Description        *string `url:"description,omitempty" json:"description,omitempty"`
+	SourceBranch       *string `url:"source_branch,omitempty" json:"source_branch,omitempty"`
+	TargetBranch       *string `url:"target_branch,omitempty" json:"target_branch,omitempty"`
+	Labels             *string `url:"labels,omitempty" json:"labels:omitempty"`
```

Comment:
@svanharmelen I just saw on other places, labels are done like this:

    Labels Labels `url:"labels,comma,omitempty" json:"labels,omitempty"`

Do you think this should be changed here to this, too?

Reply:
... I think so, see #440